libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/RouteConstants.java

```diff
@@ -48,4 +48,5 @@
   static final String SEVERE_CONGESTION_VALUE = "severe";
   static final String ORIGIN_MARKER_NAME = "originMarker";
   static final String DESTINATION_MARKER_NAME = "destinationMarker";
+  static final String MAPBOX_LOCATION_ID = "mapbox-location";
 }
```

Comment:
Minor thing: Could we do some _Boy Scout_ clean up here? There are some dead constants (never used) in `RouteConstants`.